### PR TITLE
Added english language

### PR DESCRIPTION
**Pull request by AmirHosein-Gharaati**

### CODE REVIEW

**Review by smrm-dev — APPROVED** [review]
Thanks for your pull request.

**Review by smrm-dev — DISMISSED** [review]
Thanks for your PR.

File: lib/main.dart [hunk 1]
```diff
@@ -9,13 +9,15 @@ import 'package:noteif/helper/utils.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 import 'theme/themeModeChanger.dart';
 import 'package:provider/provider.dart';
+import 'Languages.dart';
```
Comment (mrmousavi78) [inline lib/main.dart:12]: Manage your new files in new folders with proper names.

Comment (smrm-dev) [inline lib/main.dart:12]: Manage your new files in new folders with proper names.

File: lib/main.dart [hunk 2]
```diff
@@ -120,6 +129,9 @@ class _HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
 
+    Locale local = DemoLocalizations.of(context).locale;
```
Comment (smrm-dev) [inline lib/main.dart:132]: For this issue, it is better to have a button or something like that so that the user could change the language manually

**Review by smrm-dev — CHANGES_REQUESTED** [review]
Thanks for your PR.
After reviewing your PR I think you should make some changes in your PR and send it again.
Please provide a proper description for your pull request, for example, mentioning the issue you wanted to fix(#14).
You can see other PR examples in this repo for more information about sending a PR.

**Pull request closed without merging**